lib/trento/application/usecases/clusters/clusters.ex

```diff
@@ -53,7 +53,7 @@ defmodule Trento.Clusters do
   @spec get_all_clusters :: [ClusterReadModel.t()]
   def get_all_clusters do
     from(c in ClusterReadModel,
-      order_by: [asc: c.name],
+      order_by: [asc: c.name, asc: c.id],
```

Comment:
This is required in order to have a predictable order of clusters in the view, as the clusters might have the same name sometimes (as in our demo)